pkg/cmd/ports/forward.go

```diff
@@ -59,6 +60,7 @@ var PortForwardCmd = &cobra.Command{
 				views.RenderInfoMessage(fmt.Sprintf("Port %d already in use.", port))
 			}
 			views.RenderInfoMessage(fmt.Sprintf("Port available at http://localhost:%d\n", *hostPort))
+			RenderQr(fmt.Sprintf("http://localhost:%d", *hostPort), false)
 		}
 
 		if publicPreview {
```

Comment:
@ezhil56x the QR code should be rendered for the public preview only. The localhost URL is easy to copy/paste or simply enter in the browser.